app/feature/travel-certificate/src/main/kotlin/com/hedvig/android/feature/travelcertificate/ui/AddCoInsured.kt

```diff
@@ -0,0 +1,130 @@
+package com.hedvig.android.feature.travelcertificate.ui
+
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.hedvig.android.core.designsystem.component.button.LargeContainedButton
+import com.hedvig.android.core.designsystem.component.textfield.HedvigTextField
+import com.hedvig.android.core.designsystem.material3.squircle
+import com.hedvig.android.core.designsystem.preview.HedvigPreview
+import com.hedvig.android.core.designsystem.theme.HedvigTheme
+import com.hedvig.android.core.ui.ValidatedInput
+import com.hedvig.android.core.ui.clearFocusOnTap
+import com.hedvig.android.core.ui.scaffold.HedvigScaffold
+import com.hedvig.android.feature.travelcertificate.TravelCertificateUiState
+import com.hedvig.android.feature.travelcertificate.data.TravelCertificateResult
+import hedvig.resources.R
+import kotlinx.datetime.LocalDate
+
+@Composable
+fun AddCoInsured(
+  selectedCoInsuredId: String?,
+  uiState: TravelCertificateUiState,
+  onSsnChanged: (String) -> Unit,
+  onNameChanged: (String) -> Unit,
+  onRemoveClicked: (String) -> Unit,
+  onSaveClicked: () -> Unit,
+) {
+
+  val selectedCoInsured = uiState.coInsured.input.firstOrNull { it.id == selectedCoInsuredId }
+
+  HedvigScaffold(
+    navigateUp = {},
```

Comment:
Might wanna add smth here. Now pressing the arrow top left will do nothing.

Reply:
Fixed in a PR on top of this